README.md

```diff
@@ -138,11 +138,21 @@ PULSE_HISTORY_MAX=300
 
 ### Installation
 1. Install Go according to the installation instructions here: http://golang.org/doc/install
-2. Clone the repo
+
+There are two options to run Pulse:
+
+a) Clone the repo
+
    ```sh
-   https://github.com/gignative-solutions/ba-p2p-headers.git
-   ```
-    
+  git clone https://github.com/bitcoin-sv/pulse
+   ``` 
+   1. ```go run ./cmd/main.go```
+
+b) Get package from ``pkg.dev.go``
+1. ```go get -u https://pkg.go.dev/github.com/bitcoin-sv/pulse```
+2. ```go build -o pulse```
+3. ```./pulse```
```

Comment:
I'm not sure if we really want to suggest this as an option

Reply:
Pulse is one pkg.go.dev. At least v6.0.0. What your concerns ?